app/templates/celebrity.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<div class="container mt-4">
+    <div class="my-4">
+        <h5>Celebrities born today ({{ date }}):</h5>
+    </div>
+    <div class="col-lg-12">
+        <div id="celebCards" class="card-deck" style="flex-flow: row wrap; display: flex;">
+            <div class="spinner-border" role="status">
+                <span class="sr-only"></span>
+              </div>
+        </div>
+    </div>
+</div>
+<link rel="stylesheet" href="{{ url_for('static', path='/celebrity.css') }}">
+<script type="text/javascript" src="{{ url_for('static', path='/celebrity.js')}}"></script>
+<script>
```

Comment:
Move to the js file :)

Reply:
I'm passing a Python customized date variable (**{{date | safe}}**) from the Jinja2 template to the JS file, and I can't do that directly from the JS file (also, I find this method very friendly). How do you suggest doing so without adding more code? Thank you in advance!